print-battleground-state-changes

```diff
@@ -426,9 +428,8 @@ for rows in records.values():
 
 # Pull out the battleground state summaries
 battlegrounds_summarized = {
-    state: results
-    for state, results in summarized.items()
-    if state in BATTLEGROUND_STATES
+    state: summarized[state]
+    for state in BATTLEGROUND_STATES
 }
 states_updated = [
```

Comment:
This should still work but can we update this one too?

Reply:
Fixed. I didn't change `BATTLEGROUND_STATES` back to a `set` because then we'd need to worry about the order of states in `battlegrounds_summarized` (and it's small enough that `list.__contains__` will be fast enough).